cluster/src/main/scala/org/apache/spark/sql/streaming/SnappyStreamingContextFactory.scala

```diff
@@ -84,7 +84,15 @@ class SnappyStreamingContextFactory extends SparkContextFactory {
       // If Job class directly refers to any jars which has been provided
       // by install_jars, this can help.
       override def makeClassLoader(parent: ContextURLClassLoader): ContextURLClassLoader = {
-        SnappyUtils.getSnappyContextURLClassLoader(parent)
+        val cl = SnappyUtils.getSnappyContextURLClassLoader(parent)
```

Comment:
lead.urlclassloader can be passed as the parent to SnappyUtils.getSnappyContextURLClassLoader(parent) ?

Reply:
Hmm.. May be will change and see. Because parent will already = lead.uelclassloader. and in that case additional addUrl won't be required.